### DISCUSSION TITLE
LLM-Text-Loop append each answer to keep each feature

### DISCUSSION THREAD

**Discussion by xlinx** [xlinx/sd-webui-decadetw-auto-prompt-llm#14] [Show and tell]
2.Append each. ==> u will get every line llm-answer separately.
when u ask llm looply. each ask is a feature; if each is important or feture that u want show indenprnt, check it.

- example: 
- if u use red yellow blue ask llm 3-times.
- u will get mix result. like fig1
- enable append each.
- slide 1-5 how many times ask 
- **u will get 3-feature separately**

<table>
<tr>
    <td>
<img src='https://github.com/user-attachments/assets/94a246c0-28dc-44a1-849e-400d2048c311'/>
    </td>
    <td>
<img src='https://github.com/user-attachments/assets/42a2a25b-0be7-42c1-85dc-eca78ebdc10c'/>
    </td>
 </tr>
</table>
<table>
<tr>
    <td>
<img src='https://raw.githubusercontent.com/xlinx/sd-webui-decadetw-auto-prompt-llm/main/images/llm-loop-ex-1.png'/>
    </td>
    <td>
<img src='https://raw.githubusercontent.com/xlinx/sd-webui-decadetw-auto-prompt-llm/main/images/llm-loop-ex-2.png'/>
    </td>
    <td>
<img src='https://raw.githubusercontent.com/xlinx/sd-webui-decadetw-auto-prompt-llm/main/images/llm-loop-ex-4.png'/>
    </td>
 </tr>
</table>

**Comment by sandner-art** [comment]
I think this should add text into the [LLM-Your-Prompt]? Or send a separate call? It does not seem to work as it is, sometimes it just adds the keyword on the line to the prompt (i.e. "red", before the generated prompt). I am using Forge and xtuner/llava-llama-3-8b-v1_1-gguf/llava-llama-3-8b-v1_1-int4.gguf (with vision), can it have an impact?
